app/logic/userManagement.py

```diff
@@ -53,7 +53,8 @@ def changeProgramInfo(newProgramName, newContactEmail, newContactName, newLocati
 def getAllowedPrograms(currentUser):
     """Returns a list of all visible programs depending on who the current user is."""
     if currentUser.isCeltsAdmin:
-        return Program.select().order_by(Program.programName)
+        # Beans: ask if this where clause is an acceptable way to filter out the celts sponsored event from appearing in the list of programs
+        return Program.select().where(Program.programName != "CELTS-Sponsored Event").order_by(Program.programName)  
```

Comment:
We can just change this to use the id 9, right? Also, why are we not letting admins see these?

Reply:
Thanks for the suggestion about the class variable. We are not letting admins see these because the buttons that now create other celts sponsored events is returned from getAllowedTemplates. If we allowed this to return the celts sponsored events as a program, that button would appear twice on the page. Since this function is only used to, from the docstring, "Returns a list of all visible programs", we didn't see a need to return this as a program. This is the only use of this function.